qa/tasks/cephfs/caps_helper.py

```diff
@@ -90,6 +90,41 @@ def _gen_mds_cap_str(perm, fsname=None, cephfs_mntpt='/'):
     return mds_cap
 
 
+def get_mon_cap_from_keyring(keyring):
```

Comment:
Which other test(s) plans to use this helper?

Reply:
Right now none. But these methods do something that is very very generic and not something very specific to class CapTester. So it's a bad idea to keep these methods in class CapTester.

Reply:
Then these can be moved to filesystem.py?

Reply:
Yeah, but why filesystem.py? This not related to creation or management of CephFS...

Reply:
Because its _related_ to ceph file system, and you can always wrap them under a class.

Reply:
It's related to keyring and parsing keyring and not CephFS.